renovate.json

```
@@ -74,7 +74,7 @@
       "description": "Update yarn version (won't work because we need to download the script but gives an alert at least)",
       "fileMatch": ".yarnrc.yml",
       "matchStrings": [
-        "yarnPath: \\.yarn\\/releases\\/yarn-(?<currentValue>.*?)\\.cjs"
+        "yarnPath: \\.yarn\/releases\/yarn-(?<currentValue>.*?)\\.cjs"
```

We can make it broader here too just to avoid having to handle escapes

```suggestion
        "yarn\/releases\/yarn-(?<currentValue>.*?)"
```

might be enough wdyt